etc/bitlbee.profile

```diff
@@ -16,16 +16,16 @@ include /etc/firejail/disable-programs.inc
 netfilter
 no3d
 nonewprivs
-private
-private-dev
-protocol unix,inet,inet6
-seccomp
 nosound
 novideo
+protocol unix,inet,inet6
 read-write /var/lib/bitlbee
```

Comment:
Would `read-write` make more sense somewhere else in the profile? Maybe earlier like where we're putting blacklists (see the Bibletime profile for reference) since the format is similar to that of blacklists?